app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs

```diff
@@ -1,5 +1,17 @@
 <div class="ui ordered list">
-  {{#each record.eventRoles as |eventRole|}}
-    <div class="item">{{eventRole.name}} ({{eventRole.event.name}})</div>
+  {{#each record.organizerEvents as |event|}}
+    <div class="item">{{event.id}}{{t ' - Organizer'}}</div>
```

Comment:
@niranjan94  I have a doubt here. Like I am currently printing event.id, it gives the correct ids of the events for which the user is an organizer or co organizer. But if I try to print event.name/identifier or other information nothing is printed. Am I missing out something?